seatsio-android-sample/src/main/java/test/ShowSeatingChartActivity.java

```diff
@@ -70,13 +66,15 @@ protected void onCreate(Bundle savedInstanceState) {
                 .setOnHoldFailed((objects, ticketTypes) -> Log.i(LOG_PREFIX, "Hold failed " + objects))
                 .setOnReleaseHoldSucceeded((objects, ticketTypes) -> Log.i(LOG_PREFIX, "Release hold succeeded " + objects))
                 .setOnReleaseHoldFailed((objects, ticketTypes) -> Log.i(LOG_PREFIX, "Release hold failed " + objects))
-                .setObjectLabel("object => object.labels.own")
-                .setPriceFormatter(price -> price + "€")
+                .setObjectLabelJavascriptFunction("(object, defaultLabel, extraConfig) => object.labels.own")
+                .setObjectIconJavascriptFunction("(object, defaultIcon, extraConfig) => defaultIcon")
+                .setPriceFormatter("price => '€' + price")
```

Comment:
Price formatter is executed outside the iFrame, so it's fine to keep it as a Java function